neo4j-to-neptune/docs/convert-csv.md

```diff
@@ -1,74 +1,141 @@
     NAME
             neo4j-to-neptune.sh convert-csv - Converts CSV file exported from Neo4j
-            via 'apoc.export.csv.all' to Neptune Gremlin import CSV files
-    
+            via 'apoc.export.csv.all' to Neptune Gremlin load data formatted CSV files,
+            and optionally automates the bulk loading of the converted data into Amazon Neptune.
+
     SYNOPSIS
-            neo4j-to-neptune.sh convert-csv {-d | --dir} <outputDirectory>
-                    {-i | --input} <inputFile> [ --infer-types ]
+            neo4j-to-neptune.sh convert-csv [ --bucket-name <bucketName> ]
+                    [ --bulk-load ] {-d | --dir} <outputDirectory>
+                    {-i | --input} <inputFile> [ --iam-role-arn <iamRoleArn> ]
+                    [ --infer-types ] [ --monitor ]
+                    [ --neptune-endpoint <neptuneEndpoint> ]
                     [ --node-property-policy <multiValuedNodePropertyPolicy> ]
+                    [ --parallelism <parallelism> ]
                     [ --relationship-property-policy <multiValuedRelationshipPropertyPolicy> ]
+                    [ --s3-prefix <s3Prefix> ]
                     [ --semi-colon-replacement <semiColonReplacement> ]
-    
+
     OPTIONS
+            --bucket-name <bucketName>
+                S3 bucket name for CSV files to be stored
+
+                This option may occur a maximum of 1 times
+
+
+                This option is required if any of the following options are
+                specified: --bulk-load
+
+
+            --bulk-load
+                Enable bulk load to Neptune. If true, the output will be uploaded
+                to S3 and loaded into Neptune using the bulk loader (default:
+                false)
+
+                This option may occur a maximum of 1 times
+
+
             -d <outputDirectory>, --dir <outputDirectory>
                 Root directory for output
-    
+
                 This option may occur a maximum of 1 times
-    
-    
+
+
                 This options value must be a path to a directory. The provided path
                 must be readable and writable.
-    
-    
+
+
             -i <inputFile>, --input <inputFile>
                 Path to Neo4j CSV file
-    
+
                 This option may occur a maximum of 1 times
-    
-    
+
+
                 This options value must be a path to a file. The provided path must
                 exist on the file system. The provided path must be readable and
                 writable.
-    
-    
+
+
+            --iam-role-arn <iamRoleArn>
+                IAM role ARN for Neptune bulk loading
```

Comment:
It's probably worth a link to the bulk loader docs anytime this role is mentioned as it's not very intuitive for users unfamiliar with Neptune. Of course the relevant docs are split into 2 pages 😠
https://docs.aws.amazon.com/neptune/latest/userguide/bulk-load-tutorial-IAM-CreateRole.html
https://docs.aws.amazon.com/neptune/latest/userguide/bulk-load-tutorial-IAM-add-role-cluster.html